modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/CSharpClientCodegen.java

```diff
@@ -17,21 +18,33 @@
     private static final String NET45 = "v4.5";
     private static final String NET40 = "v4.0";
     private static final String NET35 = "v3.5";
+    // TODO: v5.0 is PCL, not netstandard version 1.3, and not a specific .NET Framework. This needs to be updated,
+    // especially because it will conflict with .NET Framework 5.0 when released, and PCL 5 refers to Framework 4.0.
```

Comment:
PCL 5.0 actually refers to .NET Framework 4.6 and higher, .NET Core 1.0 and Windows Universal 10.0. I commented this correctly later in the file. As a TODO, I don't think it's a huge issue.

I'd like to straighten this out before 2.3.0 release.

Reply:
Yup, 2.3.0 is a release with breaking changes so please feel free to fix these wordings before the release.